client/src/views/users/UserContextMenu.vue

```diff
@@ -54,16 +54,30 @@ export enum UserAction {
 </script>
 
 <script setup lang="ts">
-import { IonContent, IonItem, IonLabel, IonList, popoverController, IonIcon, IonItemGroup } from '@ionic/vue';
-import { personRemove, informationCircle } from 'ionicons/icons';
+import UserDetailsModal from '@/views/users/UserDetailsModal.vue';
+import { IonContent, IonIcon, IonItem, IonItemGroup, IonLabel, IonList, modalController, popoverController } from '@ionic/vue';
+import { informationCircle, personRemove } from 'ionicons/icons';
+import { UserInfo } from '@/parsec';
 
-defineProps<{
-  isRevoked: boolean;
+const props = defineProps<{
+  user: UserInfo;
 }>();
 
 function onClick(action: UserAction): Promise<boolean> {
   return popoverController.dismiss({ action: action });
 }
+
+async function presentDetailsModal(): Promise<void> {
```

Comment:
Why did you change how the context menu handles a click on `Details`? Everything is already done, you just need to add the modal here https://github.com/Scille/parsec-cloud/blob/d3e0b0536348e384a0082ef828ba77160b82cd21/client/src/views/users/ActiveUsersPage.vue#L329 and here https://github.com/Scille/parsec-cloud/blob/d3e0b0536348e384a0082ef828ba77160b82cd21/client/src/views/users/RevokedUsersPage.vue#L183

Granted, it's done twice instead of one but:
- it's temporary as we want to redo the users page eventually and group the active and revoked user pages
- it's better to let the page handle the logic rather than the submenu
- it's not a lot a code